### ISSUE TITLE
auto update webui on install

### ISSUE THREAD

**Issue by adamwbb** [hexparrot/mineos-node#256]
Future ISO installs should automatically fetch the latest webui.

I have noticed a big drawback to using the turnkey appliance is that it uses a out of date UI which causes issues when trying to use certain minecraft jars (aka Paperspigot)

**Comment by hexparrot** [comment]
That really shouldn't be considered a drawback to using Turnkey, though, should it?  If you installed MineOS manually today, it'd be out of date one commit from now, and that shouldn't outweigh the value of the distro, I think.

Fair idea though, and the next time an ISO needs to be released I'll probably put this into effect.

**Comment by flareofghast** [comment]
It can come with most up to date at time of release and have an option to update. 

There may be those who have a local only setup and you'd want to have a version pre-installed.

**Comment by hexparrot** [comment]
Tried figuring this out, but couldn't come up with a clean solution.

Right now, as before, users will be getting a local-only setup (ready to go with no external internet connection). I tried finding a place in the boot sequence where it would auto update the latest commit, but couldn't find any place where that could happen.  

It seemed like it belonged in [firstboot.d](https://github.com/hexparrot/mineos-tkldev/tree/master/overlay/usr/lib/inithooks/firstboot.d) but all of that stuff happens prior to network setup as well (even in the event that a user _does_ use DHCP).

Basically it means the only window for updating is a) if I can figure out how to modify the Turnkey Curses menu -- not seeming too possible right now, b) upon _second_ reboot, but then I don't know how to 1) ensure it happens before the service starts and 2) make sure it only happens on the first install.

It looks as if at this point, it's going to just be that all users will start off at a baseline area (e.g., a recent commit--I do them not too infrequently) and all smaller, minor commits will be not-utilized until the user goes through his or her own first update (which I encourage users to get familiar with ALL the time, anyway).

TL;DR: couldn't figure out a clean place for updating this repo to happen before service starting and after networking configuration.  Looks undoable at this time.
